test/functional/editions_controller_test.rb

```diff
@@ -200,4 +200,66 @@ class EditionsControllerTest < ActionController::TestCase
       end
     end
   end
+
+  context "edit" do
+    setup do
+      @answer = FactoryBot.create(:answer_edition)
+    end
+
+    should "render the page ok" do
+      get :show, params: { id: @answer.id }
+      assert_response :ok
+    end
+
+    context "#update" do
+      should "show update and success message and render show template when saved" do
+        get :update, params: {
```

Comment:
I'm surprised this works as a `get`—I'd have expected an update to be a `post` request.

Reply:
Yes that's a fair point. I've updated all of these requests to `post` for correctness.